datenguide_python/output_transformer.py

```diff
@@ -80,7 +80,13 @@ def build_and_merge_data_region_query(self) -> pd.DataFrame:
                 "fake_id", axis=1
             )
 
-        return data_out
+        cols_4 = [x.replace("_value", "") for x in data_out.columns]
+        data_out.columns = cols_4
+        cols_5 = ["id", "name", "year"]
+        cols_6 = [col for col in data_out.columns if col not in cols_5]
+        output_ordered = data_out.loc[:, cols_5 + cols_6]
```

Comment:
I think the ordering is missing the part where the columns containing the statistics information are all on the left as well. An easy way to achieve this might be to identify and rearrange the columns containing "_value" towards the left
and only in the last step drop the value postfix.